### PR TITLE
feat(i18n): galician language contribution

### PR DESCRIPTION
**Pull request by xmgz**
related to https://github.com/elk-zone/elk/pull/1416

### CODE REVIEW

**Comment by xmgz** [comment]
:face_with_head_bandage:  did I brake anything? this PR is getting harder to commit. I'm trying to do my best. Forked, created files, adding i18n entry, save and PR.

**Comment by userquin** [comment]
> 🤕 did I brake anything? this PR is getting harder to commit. I'm trying to do my best. Forked, created files, adding i18n entry, save and PR.

in this entru `agora|fai {n} segundo}|fai{n} segundos` remove the closing brace after `segundo}|` => `segundo|`

File: config/i18n.ts [hunk 1]
```diff
@@ -118,6 +118,11 @@ const locales: LocaleObjectData[] = [
     file: 'fi-FI.json',
     name: 'Suomea',
   },
+    {
```
Comment (userquin) [inline config/i18n.ts:121]: remove 2 spaces from here (it has 4 spaces), should be 2, you can run `pnpm lint` to check there are no errors

**Comment by userquin** [comment]
so, can we close #1416  ?

**Review by userquin — APPROVED** [review]
This PR adds a variant of the Spanish language spoken in Galicia: Galician

There are 3 other official variants of languages spoken in Spain: Catalan (Catalonia), Valencian (Valencia) and Euskera (Basque Country)

**Review by patak-dev — APPROVED** [review]

**Pull request merged**